content/en/releases/platforms.md

```diff
@@ -0,0 +1,307 @@
+| kube-proxy              |                                       |                                       | [binary_file]<br />docker_tag<br />tar<br />sha256<br />sha512 | [binary_file]<br />docker_tag<br />tar<br />sha256<br />sha512 | [binary_file]<br />docker_tag<br />tar<br />sha256<br />sha512 | [binary_file]<br />docker_tag<br />tar<br />sha256<br />sha512 |                             | exe<br />sha256<br />sha512 |
+| kube-scheduler          |                                       |                                       | [binary_file]<br />docker_tag<br />tar<br />sha256<br />sha512 | [binary_file]<br />docker_tag<br />tar<br />sha256<br />sha512 | [binary_file]<br />docker_tag<br />tar<br />sha256<br />sha512 | [binary_file]<br />docker_tag<br />tar<br />sha256<br />sha512 |                             |                             |
+| kubeadm                 |                                       |                                       | [binary_file]<br />sha256<br />sha512                          | [binary_file]<br />sha256<br />sha512                          | [binary_file]<br />sha256<br />sha512                          | [binary_file]<br />sha256<br />sha512                          |                             | exe<br />sha256<br />sha512 |
+| kubectl                 | [binary_file]<br />sha256<br />sha512 | [binary_file]<br />sha256<br />sha512 | [binary_file]<br />sha256<br />sha512                          | [binary_file]<br />sha256<br />sha512                          | [binary_file]<br />sha256<br />sha512                          | [binary_file]<br />sha256<br />sha512                          | exe<br />sha256<br />sha512 | exe<br />sha256<br />sha512 |
+| kubelet                 |                                       |                                       | [binary_file]<br />sha256<br />sha512                          | [binary_file]<br />sha256<br />sha512                          | [binary_file]<br />sha256<br />sha512                          | [binary_file]<br />sha256<br />sha512                          |                             | exe<br />sha256<br />sha512 |
+| mounter                 |                                       |                                       | [binary_file]<br />sha256<br />sha512                          | [binary_file]<br />sha256<br />sha512                          | [binary_file]<br />sha256<br />sha512                          | [binary_file]<br />sha256<br />sha512                          |                             |                             |
+
+#### Extra files (for GCE)
```

Comment:
Can we explore deprecating these GCE files? Is there an issue for this?

Reply:
they're primarily used by kube-up, it will be a little painful though, various places expect to be able to download the tarball.

we have rarely removed artifacts, even the bash scripts

Reply:
I'm not exactly sure where we publish those files.